packages/core/quri_parts/core/estimator/sampling/estimator.py

```diff
@@ -286,3 +288,25 @@ def estimator(
         )
 
     return estimator
+
+
+def create_general_sampling_estimator(
+    total_shots: int,
+    sampler: ConcurrentSampler,
+    measurement_factory: CommutablePauliSetMeasurementFactory,
+    shots_allocator: PauliSamplingShotsAllocator,
+) -> GeneralQuantumEstimator[CircuitQuantumState, ParametricCircuitQuantumState]:
+    """Creates a :class:`GeneralEstimator` that performs sampling estimation.
```

Comment:
Thank you!!
It's a minor detail, but let's make the class name more accurate.

```suggestion
    """Creates a :class:`GeneralQuantumEstimator` that performs sampling estimation.
```

Also you can find `GeneralEstimator` in `core/estimator/__init__.py` and `tests/core/estimator/test_estimator_creation.py`.

Reply:
Thank you for the review! I have corrected it.